test/Sanitizers/tsan/objc_async.swift

```diff
@@ -7,8 +7,6 @@
 // REQUIRES: concurrency
 // REQUIRES: objc_interop
 // REQUIRES: tsan_runtime
-// UNSUPPORTED: OS=linux-gnu
-// UNSUPPORTED: OS=windows-msvc
```

Comment:
Since this test requires [`objc_interop`, which is only available on Apple platforms](https://github.com/apple/swift/blob/47efc380cac31760940219e90c83b3f89f589513/test/lit.cfg#L980), these lines are not needed.